sections/Discussion.tex

```diff
@@ -1 +1,46 @@
+}
+
+Overall, we see that there is a large amount of overlap between the models, which is interesting since models use different meta-information to create the various topic distributions.
+This indicates that the models share many of the top words, and indicate a slight deviation between the models due to the meta-information.
+The Doc-topic model shows words, like "farming" and "production", which is what the article is mostly about.
+This behavior is to be expected since the performance of \gls{lda} has been explored and evaluated before. 
+Author-topic specific words are not very present and are only showing three unique words: "byder", "hører" and "børnene".
+This indicates that the Author-topic model has trouble generalizing what the author of this article(Peter Tordrup Larsen) is writing about since then he has written $5002$ articles. 
+The Category topic model only shows four unique words: "søndag", "leg", "klokken" and "danskere".
+These words are also very abstract and can be used in many different scenarios.
+
+The very interesting part of this analysis is the words appearing in multiple models.
+Some of the words, within this category are: "arbejdet", "jorden", "dyr" and "norjyske".
+Combining the results these models might yield better topic models, but we can not conclude that based on only one article.
+These summarize the text pretty well, but it is also hard to summarize this text since it covers a wide variety of specific topics.

```

Comment:
```suggestion
These words summarize the text pretty well, but it is also hard to summarize this text since it covers a wide variety of specific topics.
Combining the results of these models might yield better topic models, but we can not conclude that based on only one article.
There is also the possibility that choosing another random article would give completely different numbers of marked words per model, because this highly depends on the article's author and category.
```
Byttet sætningerne om og tilføjet lidt om hvis en anden artikel blev valgt.